VaticanBAV/et/BAVet176.xml

```diff
@@ -27,12 +30,211 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        </item>
+                                        <item xml:id="q3" n="2">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="15r" to="24v"/>
+                                        </item>
+                                        <item xml:id="q4" n="3">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="25r" to="34v"/>
+                                        </item>
+                                        <item xml:id="q5" n="4">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="35r" to="42v"/>
+                                        </item>
+                                        <item xml:id="q6" n="5">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="43r" to="50v"/>
+                                        </item>
+                                        <item xml:id="q7" n="6">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="51r" to="60v"/>
+                                        </item>
+                                        <item xml:id="q8" n="7">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="61r" to="66v"/>
+                                        </item>
+                                        <item xml:id="q9" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="67r" to="76v"/>
+                                        </item>
+                                    </list>-->
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout writtenLines="16"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc>
+                                        A title indicating the first and third works is written in faded pencil.
+                                    </desc>
+                                    <q xml:lang="gez">
+                                        <seg>ማህ<choice><sic>ለ</sic></choice>ተ፡ ጽጌ፡</seg>
+                                        <seg>ቍስቋም፡</seg><!--ie መልክአ፡ ቍስቋም፡-->
+                                        <!--Grébaut separates the two parts with 'et', but doesn't say whether they are written together or separately-->
```

Comment:
I would suggest a note: "Grébaut separates the two parts, but doesn't say whether they are written together or separately."